virtcontainers/container_test.go

```diff
@@ -140,53 +134,37 @@ func TestContainerRemoveDrive(t *testing.T) {
 }
 
 func testSetupFakeRootfs(t *testing.T) (testRawFile, loopDev, mntDir string, err error) {
+	assert := assert.New(t)
 	if tc.NotValid(ktu.NeedRoot()) {
 		t.Skip(testDisabledAsNonRoot)
 	}
 
 	tmpDir, err := ioutil.TempDir("", "")
-	if err != nil {
-		t.Fatal(err)
-	}
+	assert.NoError(err)
 
 	testRawFile = filepath.Join(tmpDir, "raw.img")
-	if _, err := os.Stat(testRawFile); !os.IsNotExist(err) {
-		os.Remove(testRawFile)
-	}
+	_, err = os.Stat(testRawFile)
+	assert.True(os.IsNotExist(err))
 
 	output, err := exec.Command("losetup", "-f").CombinedOutput()
-	if err != nil {
-		t.Fatalf("Skipping test since no loop device available for tests : %s, %s", output, err)
-		return
-	}
+	assert.NoError(err)
 	loopDev = strings.TrimSpace(string(output[:]))
 
-	output, err = exec.Command("fallocate", "-l", "256K", testRawFile).CombinedOutput()
-	if err != nil {
-		t.Fatalf("fallocate failed %s %s", output, err)
-	}
+	_, err = exec.Command("fallocate", "-l", "256K", testRawFile).CombinedOutput()
+	assert.NoError(err)
 
-	output, err = exec.Command("mkfs.ext4", "-F", testRawFile).CombinedOutput()
-	if err != nil {
-		t.Fatalf("mkfs.ext4 failed for %s:  %s, %s", testRawFile, output, err)
-	}
+	_, err = exec.Command("mkfs.ext4", "-F", testRawFile).CombinedOutput()
+	assert.NoError(err)
 
-	output, err = exec.Command("losetup", loopDev, testRawFile).CombinedOutput()
-	if err != nil {
-		t.Fatalf("Losetup for %s at %s failed : %s, %s ", loopDev, testRawFile, output, err)
-		return
-	}
+	_, err = exec.Command("losetup", loopDev, testRawFile).CombinedOutput()
```

Comment:
May be helpful to add output as msg to assert for more verbose failures; only a thought - YMMV